rnaseq/stringtie.cwl

```diff
@@ -31,4 +32,8 @@ outputs:
     gtf:
         type: File
         outputBinding:
-            glob: transcripts.gtf
+            glob: stringtie_transcripts.gtf
+    tsv:
```

Comment:
`gtf` is good inspiration for this, but `tsv` is a rather generic name! Maybe `gene_expression` (or `gene_expression_tsv`)?  (I'd probably rename the other one too!)

Reply:
Yeah, you're right. I went ahead and renamed all of these labels to something more descriptive.